### PR TITLE
[W6.6h][T10-B1]Wu Qihao

### PR DESCRIPTION
**Pull request by qihao27**
W6.6h

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -87,6 +87,39 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+### Use case: renaming an existing tag
+
+**MSS**
+
+1. User requests to rename an existing tag
```
Comment (jeffryhartanto) [inline doc/DeveloperGuide.md:95]: this one should be something like `user list existing tags`

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -87,6 +87,39 @@ Use case ends.
...
+2. AddressBook shows a list of existing tags
+3. User requests to rename a specific tag in the list to a new tag name
+4. AddressBook confirms the change with user
+5. User confirms the change
```
Comment (jeffryhartanto) [inline doc/DeveloperGuide.md:99]: Keep in mind that (especially for the project) that the app should have `advanced CLI`.

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -87,6 +87,39 @@ Use case ends.
...
+6. AddressBook renames the specified tag with the new tag name <br>
+Use case ends.
+
+**Extensions**
+
+2a. The list is empty
```
Comment (jeffryhartanto) [inline doc/DeveloperGuide.md:105]: This should be 1a. Check other extensions

**Comment by jeffryhartanto** [comment]
@qihao27 Some comments added. Please close the PR after reading comments.

Good job for doing many LOs.

File: doc/DeveloperGuide.md (hunk 1 above)
Comment (qihao27) [reply doc/DeveloperGuide.md:95]: my idea is that user input command like "change tag" and the app will list out all the existing tags used in step 2.

File: doc/DeveloperGuide.md (hunk 3 above)
Comment (qihao27) [reply doc/DeveloperGuide.md:105]: Other extension such as "Delete person" indicate this as an extension for step that "addressbook shows a list of persons"

**Pull request closed without merging**